suite.go

```diff
@@ -13,7 +13,46 @@ type Suite struct {
 	Name   string
 	Time   string
 	Status string
-	Tests  []*Test
+	tests  []*Test
+}
+
+// AddTest adds a test case to the suite
+func (suite *Suite) AddTest(test *Test) {
```

Comment:
This change is part of a bigger local refactoring branch. Unfortunately, it has not come to fruition yet, so I thought I will commit and PR early.